source/agora/test/Script.d

```diff
@@ -74,11 +74,11 @@ unittest
 
     const lock_height_2 = Height(2);
     auto unlock_height_2 = block_6.spendable()
-        .map!(txb => txb.lock(lock_height_2).sign(OutputType.Payment, 0, &keyUnlocker));
+        .map!(txb => txb.unlock_signer(&keyUnlocker).lock(lock_height_2).sign(OutputType.Payment, 0));
```

Comment:
Out of curiosity, what's the reason this introduces the new function, `unlock_signer`?

Reply:
We want to calculate the fees when doing `split` so need to know the `unlocker` so that the size of the `input`s can be calculated. Previously it was passed as a param to `sign`.